Makefile

```diff
@@ -2,10 +2,11 @@ ENV ?= staging
 
 apply: apply-remote-state apply-secrets apply-env apply-k8s-utils
 
+## remove state file only if exit code 0 from terraform apply
 apply-remote-state:
 	pushd terraform/bootstrap/remote-state; \
 	terraform init; \
-	terraform apply -var "environment=$(ENV)"
+	terraform apply -var "environment=$(ENV)" && rm ./terraform.tfstate;
 
 apply-secrets:
 	pushd terraform/bootstrap/secrets; \
```

Comment:
running into some trouble with the teardown command for secrets, since it was changed to a name prefix, ill need to get the nameback to delete it 😢